1.7/administration/installing/custom/system-requirements/install-docker-centos.md

```diff
@@ -0,0 +1,53 @@
+---
+post_title: Install Docker on CentOS
+menu_order: 2
+---
+Docker's <a href="https://docs.docker.com/engine/installation/linux/centos/" target="_blank">CentOS-specific installation instructions</a> are always going to be the most up to date for the latest version of Docker. However, because of the combination of using systemd and OverlayFS, it may be simpler to follow the following condensed steps.
+
```

Comment:
I'd like to be more precise here: why do systemd and OverleyFS need to be combined? Why might it be simpler to follow the steps here?